ipa-core/src/ff/prime_field.rs

```diff
@@ -241,6 +242,15 @@ macro_rules! field_impl {
             }
         }
 
+        // If the field value fits in a machine word, a naive comparison should be fine.
+        // But this impl is important for `[T]`, and useful to document where a
+        // constant-time compare is intended.
+        impl ConstantTimeEq for $field {
```

Comment:
Do you think it makes sense to make the Eq implementation for the `field_impl` macro default constant time as well?